.ci/docker-compose-file/docker-compose-elastic-search-tls.yaml

```diff
@@ -54,7 +56,7 @@ services:
       - xpack.security.http.ssl.certificate=certs/es01/es01.crt
       - xpack.security.http.ssl.certificate_authorities=certs/ca/ca.crt
       - xpack.license.self_generated.type=${LICENSE}
-    mem_limit: 1073741824
+    mem_limit: 4G
```

Comment:
I remember I need 2G heap for jvm just to start it 5 years ago.

Reply:
🫠
I guess the new "features" now require 4 G :sweat_smile: